src/core/Util.js

```diff
@@ -239,3 +239,19 @@ export function cancelAnimFrame(id) {
 		cancelFn.call(window, id);
 	}
 }
+
+function isNumeric(str) { // https://stackoverflow.com/a/175787/2520247
+	return !isNaN(str) && !isNaN(parseFloat(str));
+}
+
+// keep it private intentionally, because such function implementation
+// heavily depends on it's particular application and unable to suit all the cases
+export function _checkNumber(a) {

```

Comment:
```suggestion
// @function checkNumber(data: Number|String): Number
// Makes sure that passed `data` is finite number, and returns it.
// If string is passed - converts it to number trying to avoid coercion pitfalls.
// Throws error in case of any failure.
export function checkNumber(a) {
```

@IvanSanchez
But I'm not sure, because fixing it in public API
- .. makes impossible future internal changes. E.g. what if later we'll decide to throw on strings like '1e9'
- .. opens the door to misusing (remember using `Browser.touch`)

Reply:
@IvanSanchez what do you think about that points? I personally would like to have a public function so it can be easy used in plugins. Not everyone should reinvent the wheel.

Reply:
@mourner ?

Reply:
@mourner what do you think is better? A public or private function for checking if the input is a number.